history/taskana-simplehistory-provider/src/main/java/pro/taskana/simplehistory/TaskanaHistoryEngine.java

```diff
@@ -10,4 +12,20 @@
    * @return the HistoryService
    */
   TaskanaHistory getTaskanaHistoryService();
+
+  /**
+   * check whether the current user is member of one of the roles specified.
+   *
+   * @param roles The roles that are checked for membership of the current user
+   * @return true if the current user is a member of at least one of the specified groups
+   */
+  boolean isUserInRole(TaskanaRole... roles);
```

Comment:
Don't we have access to the engine to reuse the existing methods?

Reply:
unfortunately no...as soon as we pass the engine in the spi, this will be different and should be changed with the upcoming ticket to make the whole process transactionally safe